.github/workflows/validate-documentation.yml

```diff
@@ -7,19 +7,30 @@ on:
     - '**.md'
     - 'package-lock.json'
     - 'package.json'
+    - '.lycheeignore'
   pull_request:
     branches: [ main ]
     paths:
     - '**.md'
     - 'package-lock.json'
     - 'package.json'
+    - '.lycheeignore'
+  workflow_dispatch:
+    paths:
+    - '**.md'
+    - 'package-lock.json'
+    - 'package.json'
+    - '.lycheeignore'
```

Comment:
why have you add `paths` to `workflow_dispatch`? Does it do anything? I think this should be removed.

Reply:
c77071a9ccf2e26dd341e65937a37e8b1dd11d22